Application/EdFi.Ods.Api/Controllers/DataManagementControllerBase.cs

```diff
@@ -342,7 +342,7 @@ protected string GetResourceUrl(Guid id)
                         urlBuilder.Port = Request.Host.Port.Value;
                     }
 
-                    _applicationUrl = $"{urlBuilder.Uri}{GetResourceCollectionName()}";
```

Comment:
With the removal of this method, does it make sense to fix codegen and remove it from the templates? @vimayya @blmeyers

Reply:
Yeah, if its not needed drop the abstract version in DataManagementControllerBase and clean up the generated code.

Reply:
I have cleaned up the code. Thanks